SoftFever/OrcaSlicer
# Potential fix for other layers bed temperature not applied

# Description

On occasion the second layer bed temperature was not set according to the filament preferences. This PR aims to introduce a fix for this. This issue seems to have been present since v1.6.4 or earlier.

This is a particularly difficult one to reproduce as it doesn't happen repeatably.

Requesting community testers to validate the fix. 

**Test needed:**
1. Slice a model with a filament that needs a different temperature for the second layer
2. Change the brim setting to auto and slice - confirm that the second layer bed temperature is being emitted
3. Change the brim setting back to none and confirm that the second layer bed temperature is emitted 

To see the Gcode window press key "C" on your keyboard. Navigate using the sliders to the second layer and at the start of it as shown in the screenshots below.

Asking for testers as its been particularly difficult to a) replicate that the issue is happening as its sporadic and b) validate that the fix is working. 

When testing please confirm with your existing installation that the issue is happening first before trying the PR build. Please comment back below on whether a) it happened to you in the current build you're using alongside the version number you're running and b) if this PR fixed it or not.

**When it works it should look like the below (line 774)**
![image](https://github.com/SoftFever/OrcaSlicer/assets/59056762/fb502a04-1945-4e23-8a62-e9348ba0e84f)

**If it doesn't work it will look like this (line 773 only, no set bed temperature present)**
![image](https://github.com/SoftFever/OrcaSlicer/assets/59056762/a6a2b04d-8865-4096-a1f3-f175c851051c)

**Tests performed**
As above.
I think that this PR also causes the bed temperature to always be set on the second layer but this also needs testing... Whether this is an issue or not remains open for discussion as its an extra Gcode command that is not needed but not harmful as the printer is already operating at that bed temperature so it will be just ignored.

Fixes #5485 
Fixes #4676
Fixes #3015
Fixes #1853
